source/opt/inline_pass.h

```diff
@@ -117,11 +134,31 @@ class InlinePass : public Pass {
                      ir::UptrVectorIterator<ir::Instruction> call_inst_itr,
                      ir::UptrVectorIterator<ir::BasicBlock> call_block_itr);
 
-  // Returns true if |inst| is a function call that can be inlined.
+  // Return true if |inst| is a function call that can be inlined.
   bool IsInlinableFunctionCall(const ir::Instruction* inst);
 
-  // Returns true if |func| is a function that can be inlined.
-  bool IsInlinableFunction(const ir::Function* func);
+  // Compute structured successors for function |func|.
+  // A block's structured successors are preceded by its
+  // merge block if its a header. This assures correct depth
+  // first search in the presence of early returns and kills.
```

Comment:
Assuming the mention of early returns here is apt, isn't OpKill an just early return from the entire shader? OpUnreachable is similar in that it has no predecessor, but since it isn't reachable, it was not clear that it causes a problem for DepthFirstSearch. If you think it should be mentioned, I am open to it.